src/main/java/seedu/address/logic/commands/TagListCommand.java

```diff
@@ -0,0 +1,43 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+
+/**
+ * Tags a user with one or more tags.
+ */
+public class TagListCommand extends TagCommandGroup {
```

Comment:
should the naming follow the format used in other tag command classes?

Reply:
Thanks I changed it